bin/nimbus-fml.sh

```diff
@@ -167,11 +167,11 @@ else
     # Otherwise, we should download a pre-built copy from github.com/mozilla/application-services/releases
     FML_DIR="$NIMBUS_DIR/bin"
     export BINARY_PATH="$FML_DIR/nimbus-fml"
-    AS_DOWNLOAD_URL="https://firefox-ci-tc.services.mozilla.com/api/index/v1/task/project.application-services.v2.nimbus-fml.$AS_VERSION/artifacts/public%2Fbuild%2F"
```

Comment:
I'm filing a ticket for this, but the script should be smart enough to check both github for a stable release and taskcluster for a nightly one.

Or alternatively, find a stable home for both nightly and release artifacts. cc @bendk

Reply:
For release 115 and forward, I believe the location will be:
 - archive.mozilla.org for releases
 - taskcluster for nightlies

I'll make sure to update the rust-components-swift and firefox-ios code once we're there.  I was hoping to do this for the 114 release, but I didn't make the deadline.